src/components/SliderCard/SliderCard.js

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import { Button, Card, ProgressBar } from "react-bootstrap";
+
+import "./SliderCard.css";
+
+class SliderCard extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      name: "",
+      score: 0,
+      image: ""
+    };
+  }
+
+  componentDidMount() {
+    fetch(`https://api.teleport.org/api/urban_areas/slug:${this.props.city}/`)
+      .then(response => response.json())
+      .then(response => {
+        console.log(response);
```

Comment:
`console.log` should only be used during debug, you can remove these calls before committing your code.